packages/autorest.typescript/CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 6.0.22 (2024-04-30)
+
+- [Feature] Generate types for new MFD design. Please refer to [#2455](https://github.com/Azure/autorest.typescript/pull/2455).
```

Comment:
Can we use multipart/form-data in changelog?  This should be a more official term to use.

Reply:
updated.